tests/integration/targets/aci_criterion/tasks/main.yml

```diff
@@ -0,0 +1,182 @@
+# Test code for the ACI modules
```

Comment:
This module is not needed anymore when default criterion is created as part of aci_epg. So commits can be dropped that include this code.

Reply:
module removed